src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingServiceTest.java

```diff
@@ -170,11 +170,11 @@ void testFilingGenerationWhenSuccessfulWithoutTrustsAndWithIdentityChecks() thro
         assertEquals(FILING_KIND_OVERSEAS_ENTITY, filing.getKind());
         assertEquals(FILING_DESCRIPTION_IDENTIFIER, filing.getDescriptionIdentifier());
         assertEquals("Filing Description with registration date 26 March 2022", filing.getDescription());
+        assertEquals("Joe Bloggs Ltd", filing.getData().get("entity_name"));
         final PresenterDto presenterInFiling = (PresenterDto)filing.getData().get("presenter");
         assertEquals("Joe Bloggs", presenterInFiling.getFullName());
         assertEquals("user@domain.roe", presenterInFiling.getEmail());
         final EntityDto entityInFiling = ((EntityDto) filing.getData().get("entity"));
-        assertEquals("Joe Bloggs Ltd", entityInFiling.getName());
```

Comment:
Same with these - revert. You only need to worry about adding the new field to the top-level object and nothing related to removing the field from 'entity'.

Reply:
Reverted